src/components/NavigationBar/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link, useLocation } from 'react-router-dom';
+import classnames from 'classnames';
+import { useFlags } from 'launchdarkly-react-client-sdk';
+
+import './index.scss';
+
+type NavigationProps = {
+  mobile?: boolean;
+  signout?: () => void;
+  userName?: string;
+};
+
+export const navLinks = [
+  {
+    link: '/',
+    label: 'home'
+  },
+  {
+    link: '/system',
+    label: 'systems'
+  },
+  {
+    link: '/system/making-a-request',
+    label: 'addSystem'
+  },
+  {
+    link: '/508/making-a-request',
+    label: 'add508Request'
+  },
+  {
+    link: '/help',
+    label: 'help'
+  }
+];
+
+const NavigationBar = ({ mobile, signout, userName }: NavigationProps) => {
+  const { t } = useTranslation();
+  const location = useLocation();
+  const flags = useFlags();
+
+  const responsiveContainerClass = classnames('grid-container', {
+    'navbar--container': !mobile
+  });
+
+  const responsiveCurrentClass = (current: boolean) =>
+    classnames('easi-nav__item', {
+      'easi-nav__not-current': !current
+    });
+
+  return (
+    <nav aria-label={t('header:navigation')} data-testid="navigation-bar">
+      <div className="navbar--divider" />
+      <ul className={responsiveContainerClass}>
+        {navLinks.map(route =>
+          (!flags.systemProfile && route.label === 'systems') ||
```

Comment:
I'm not thrilled by putting this logic here. I think a cleaner way to do it would be to change `navLinks` to a function that takes `flags` as an argument, then has an `isEnabled` property on each route that can depend on the flags (or is always enabled). Then down here in the render code, we can just check `route.isEnabled`.

What I've got in mind:
```
export const navLinks = (flags: Flags) => [
  {
    link: '/',
    label: 'home',
    isEnabled: true
  },
  {
    link: '/system',
    label: 'systems',
    isEnabled: flags.systemProfile
  },
  {
    link: '/system/making-a-request',
    label: 'addSystem',
    isEnabled: true
  },
  {
    link: '/508/making-a-request',
    label: 'add508Request',
    isEnabled: true
  },
  {
    link: '/help',
    label: 'help',
    isEnabled: flags.help
  }
];

// down inside `return()`:
{navLinks(flags).map(
          route =>
            route.isEnabled && /* HTML for the link */
            )
        )}
```

Reply:
You're right that makes more sense and is cleaner.  I like that suggestion.

Reply:
I had initially done exactly what you said, but I was using the navLink const as an export for unit testing.  The useFlags needs to occur inside a functional component.  Instead of duplicating that navLink in the code and unit testing, I exported it outside the component, which doesn't allow the flag values to be read.  I need to think about this a bit, to see if we can get both ideas to work

Reply:
The flag values have to be processed inside the component.  I can apply them to the navLinks upon being called.

Reply:
Dylan and I synced for a solution.  Will commit proposed changes